platform/ioscross/custom-ios.cmake

```diff
@@ -0,0 +1,13 @@
+set(USE_PORTAUDIO 0)
+set(USE_PORTMIDI 0)
+set(BUILD_STATIC_LIBRARY ON)
+set(BUILD_JAVA_INTERFACE OFF)
+set(INSTALL_PYTHON_INTERFACE OFF)
+set(BUILD_UTILITIES OFF)
+set(BUILD_TESTS OFF)
+set(HAVE_TERMCAP OFF)
+set(USE_DOUBLE 0)
+
+set(CMAKE_CXX_COMPILER $ENV{OSXCROSS_TARGET_DIR}/usr/bin/arm-apple-darwin11-clang++)
+set(CMAKE_C_COMPILER $ENV{OSXCROSS_TARGET_DIR}/usr/bin/arm-apple-darwin11-clang)
```

Comment:
Can these not be determined by Cmake? It seems a bit too machine specific

Reply:
I updated these to make them less platform specific.